src/main/java/connectingstar/tars/user/domain/User.java

```diff
@@ -0,0 +1,46 @@
+package connectingstar.tars.user.domain;
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long Id;
+
+    private String nickname; //닉네임
+
+    private String gender; //남,여,선택안함
+
+    private String ageRange; //나이 연령대
+
+    private String referrer; //유입 경로
+
+    private String identity; //정체성
+
+    private String email; //소셜 로그인 이메일
+
+    private String profileCharacter; //프로필 캐릭터
+
+    @Enumerated
+    private SocialType socialType;
```

Comment:
```suggestion
    @Enumerated(EnumType.String)
    private SocialType socialType;
```

@Enumerated 어노테이션의 기본값은 `EnumType.ORDINAL`로 Enum의 name 값을 저장하는 것이 아닌 순서를 저장하는 것으로 알고 있습니다. 
DB에는 name 값이 저장되는 것이 직관적으로 파악할 수 있고, 데이터를 조회하여 사용할 때도 name 값이 더 편할 것 같습니다.

Reply:
순서를 지정하는 것인지는 처음 알았네요. 감사합니다